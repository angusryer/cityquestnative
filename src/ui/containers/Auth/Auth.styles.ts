import { StyleSheet } from 'react-native';

const styles = StyleSheet.create({
    container: {
        height: '100vh',
        justifyContent: 'center',
        alignItems: 'center'
    }
})

export default styles;