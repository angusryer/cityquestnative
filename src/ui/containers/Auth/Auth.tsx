import React from 'react'
import { View, Text } from 'react-native'
import styles from './Auth.styles'

export default function Auth() {
    return (
        <View style={styles.container}>
            <Text>This is the Auth screen</Text>
        </View>
    )
}
