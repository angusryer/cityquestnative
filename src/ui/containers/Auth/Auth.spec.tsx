import React from "react";
import renderer from "react-test-renderer";
import Auth from "./Auth";

describe("Auth screen", () => {
    it("<Auth /> matches previous snapshot", () => {
        const tree: any = renderer.create(<Auth />).toJSON();
        expect(tree).toMatchSnapshot();
    })
})