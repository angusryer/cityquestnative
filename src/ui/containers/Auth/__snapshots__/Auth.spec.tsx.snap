// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth screen <Auth /> matches previous snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "height": "100vh",
      "justifyContent": "center",
    }
  }
>
  <Text>
    This is the Auth screen
  </Text>
</View>
`;
